content/docs/command-reference/metrics/diff.md

```diff
@@ -41,10 +41,11 @@ lists all the current metrics without comparisons.
 
 ## Options
 
-- `--targets <paths>` - limit command scope to these metric files. Using `-R`,
-  directories to search metric files in can also be given. When specifying
-  arguments for `--targets` before `revisions`, you should use `--` after this
-  option's arguments, e.g.:
+- `--targets <paths>` - limit command scope to these metric files (supports any
+  file, even when not found as `metrics` in `dvc.yaml`). Using `-R`, directories
```

Comment:
I think we should probably avoid "supports any file, even when not found as `metrics` in `dvc.yaml`" everywhere. It's better to specify at the top level that plots files are by definition could be anywhere. And `dvc.yaml` is used to get defaults (among other things)